catalog/MDCCatalog/MDCCatalogStyle.m

```diff
@@ -16,10 +16,19 @@
 
 #import "MDCCatalogStyle.h"
 
+#define SYSTEM_VERSION_GREATER_THAN_OR_EQUAL_TO(v)  ([[[UIDevice currentDevice] systemVersion] compare:v options:NSNumericSearch] != NSOrderedAscending)
```

Comment:
This is a good idea and I've seen it's very popular, but in MDC macros are discouraged whenever possible.  Please switch to a (possibly `static inline`) function using  [`-[NSProcessInfo isOperatingSystemAtLeastVersion:]`](https://developer.apple.com/documentation/foundation/nsprocessinfo/1414876-isoperatingsystematleastversion?language=objc) for runtime checks we've used in some components.  Here's a proposed solution that should work since we have a deployment target of iOS 8+:

```objectivec
static inline BOOL isRunningiOS9OrGreater() {
  static NSOperatingSystemVersion iOS9Version = {9, 0, 0};
  NSProcessInfo *processInfo = [NSProcessInfo processInfo];
  return [processInfo isOperatingSystemAtLeastVersion:iOS9Version];
}
```

This also reduces the chances of errors when typing the string literal.